module/VuFind/src/VuFind/RecordTab/HoldingsILS.php

```diff
@@ -100,13 +100,31 @@ public function getDescription()
     public function getUniqueCallNumbers($items)
```

Comment:
I would add a second `$fullDetails` parameter here to request the more detailed return format -- so if it's true, it returns an array of arrays, and if it's false, it returns plain call numbers as before. It can default to false. This way, if people have custom templates expecting the old return format, this change won't break them.

There are a few different ways you could approach this -- perhaps the least complicated (though also least efficient) would be to always build the full detailed array, but when only call numbers are requested, filter it down to just the call numbers. I have no objection to this approach if it seems best to you, but feel free to try something different if you have other ideas.